### ISSUE TITLE
App crashes

### ISSUE THREAD

**Issue by dennisje00** [kruimel0/yamaha-app-homey#10]
The app is nog working anymore

**Comment by lucasvdh** [comment]
Its still working for me though it does frequently crash and then needs to be restarted.

@dennisje00 It would be usefull if you provide more details on what exactly is not working. When you log into your Homey ([https://my.athom.com/](https://my.athom.com/)) and go to `Settings` > `Apps` there should be an alert icon next to the Yamaha app. Click on it and post that text here.

**Comment by lucasvdh** [comment]
Yep, that's it. It looks like the entire app crashes when it can't reach the receiver. Would expect some sort of error handling instead of crashing.

I am however not the maintainer of this repository. Maybe @kruimel0 has some time to look at it?
